helm/dashboards/dashboards/shared/private/persistentvolume.json

```
@@ -0,0 +1,1586 @@
+           "value": "anteater"
+         },
+         "datasource": {
+           "type": "prometheus",
+           "uid": "${datasource}"
+         },
+         "definition": "label_values(kubelet_volume_stats_capacity_bytes{app=\"kubelet\"},cluster_id)",
+         "hide": 0,
+         "includeAll": false,
+         "label": "Cluster",
+         "multi": false,
+         "name": "cluster_id",
+         "options": [],
+         "query": {
+           "query": "label_values(kubelet_volume_stats_capacity_bytes{app=\"kubelet\"},cluster_id)",
+           "refId": "PrometheusVariableQueryEditor-VariableQuery"
+         },
+         "refresh": 1,
+         "regex": "",
+         "skipUrlSync": false,
+         "sort": 0,
+         "type": "query"
+       },
+       {
+         "allValue": ".*",
+         "current": {
+           "selected": true,
+           "text": [
+             "All"
+           ],
+           "value": [
+             "$__all"
+           ]
+         },
+         "datasource": {
+           "type": "prometheus",
+           "uid": "${datasource}"
+         },
+         "definition": "label_values(kubelet_volume_stats_capacity_bytes{app=\"kubelet\"},namespace)",
+         "hide": 0,
+         "includeAll": true,
+         "label": "Namespace",
+         "multi": true,
+         "name": "namespace",
+         "options": [],
+         "query": {
+           "query": "label_values(kubelet_volume_stats_capacity_bytes{app=\"kubelet\"},namespace)",
+           "refId": "PrometheusVariableQueryEditor-VariableQuery"
+         },
+         "refresh": 2,
+         "regex": "",
+         "skipUrlSync": false,
+         "sort": 1,
+         "tagValuesQuery": "",
+         "tagsQuery": "",
+         "type": "query",
+         "useTags": false
+       },
+       {
+         "current": {
+           "selected": true,
+           "text": "Prometheus",
+           "value": "Prometheus"
+         },
+         "hide": 0,
+         "includeAll": false,
+         "label": "Datasource",
+         "multi": false,
+         "name": "datasource",
+         "options": [],
+         "query": "prometheus",
+         "queryValue": "",
+         "refresh": 1,
+         "regex": "",
+         "skipUrlSync": false,
+         "type": "datasource"
+       },
+       {
+         "current": {
+           "selected": false,
+           "text": "80",
+           "value": "80"
+         },
+         "hide": 2,
+         "label": "PVC % Used Warning Threshold",
+         "name": "warning_threshold",
+         "options": [
+           {
+             "selected": true,
+             "text": "80",
+             "value": "80"
+           }
+         ],
+         "query": "80",
+         "skipUrlSync": false,
+         "type": "textbox"
+       }
+     ]
+   },
+   "time": {
+     "from": "now-1h",
+     "to": "now"
+   },
+   "timepicker": {},
+   "timezone": "utc",
+   "title": "Kubernetes Persistent Volumes",
+   "uid": "119b92a8e8041bd567af9edab12c840c",
```

Currently, this enough to avoid conflicts with the mixins dashobard.
But I think we can do even better.

You can set human-readable UID, which will ease when you link the dashboard in alerts.
And I would emphasis in the title that this dashboard is a GS one, as opposed to the mixins one.
```suggestion
   "title": "Giantswarm / Kubernetes Persistent Volumes",
   "uid": "GSPersistentVolumes",
```

WDYT?

---

this makes sense to me 👍🏻